validation/schema_testing/officers/officerDeltaSchema_test.go

```diff
@@ -160,3 +162,31 @@ func TestOfficerDeltaSchemaNoRequestBodyError(t *testing.T) {
 		})
 	})
 }
+
+// TestOfficerDeltaSchemaMaxPropertiesError asserts that when an invalid request body is given which breaks the allowed
+// bounds on the maxProperties field for the Identification object, then an errors array is returned.
+func TestOfficerDeltaSchemaMaxPropertiesError(t *testing.T) {
+
+	Convey("Given I want to test the officers-delta API schema for maxProperty constraints", t, func() {
+
+		maxPropertiesErrorRequestBody := common.ReadRequestBody(maxPropertiesRequestBodyLocation)
+
+		r := httptest.NewRequest("POST", officersEndpoint, bytes.NewBuffer(maxPropertiesErrorRequestBody))
```

Comment:
Small one, probably worth putting POST in a const as it's used through out the file

Reply:
Implemented, nice one!